server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -714,10 +714,8 @@ public IndexSettings(final IndexMetadata indexMetadata, final Settings nodeSetti
             MergePolicyConfig.INDEX_MERGE_POLICY_MAX_MERGE_AT_ONCE_SETTING,
             mergePolicyConfig::setMaxMergesAtOnce
         );
-        scopedSettings.addSettingsUpdateConsumer(
-            MergePolicyConfig.INDEX_MERGE_POLICY_MAX_MERGE_AT_ONCE_EXPLICIT_SETTING,
-            mergePolicyConfig::setMaxMergesAtOnceExplicit
-        );
+        // todo: remove this in 2.0.0 since it was removed in lucene 9
```

Comment:
are we going to deprecate this in the next 1.x release and then remove in 2.0?

Reply:
Yes. I'm backporting this PR and open a separate remove PR to main.